lib/workers/repository/update/pr/changelog/bitbucket/bitbucket.spec.ts

```diff
@@ -0,0 +1,164 @@
+import { ChangeLogProject, ChangeLogRelease, getChangeLogJSON } from '..';
+import { Fixtures } from '../../../../../../../test/fixtures';
+import * as httpMock from '../../../../../../../test/http-mock';
+import { mocked, partial } from '../../../../../../../test/util';
+import * as semverVersioning from '../../../../../../modules/versioning/semver';
+import * as _hostRules from '../../../../../../util/host-rules';
+import type { BranchUpgradeConfig } from '../../../../../types';
+import { getReleaseNotesMd } from '../release-notes';
+
+jest.mock('../../../../../../modules/datasource/npm');
+jest.mock('../../../../../../util/host-rules');
+
+const hostRules = mocked(_hostRules);
```

Comment:
do you really need to mock hostrules? can't you simply add one for tests?

Reply:
i think you can remove all hostrule stuff from test and don't mock it at all. bitbucket http class doesn't need a token to work.